stage1/init/init.go

```diff
@@ -483,13 +467,13 @@ func stage1() int {
 		}
 	}
 
-	flavor, systemdStage1Version, err := p.getFlavor()
+	flavor, _, err := p.getFlavor()
```

Comment:
I considered it but I thought maybe we need the version in the future... Maybe I'm too conservative :P